Selenium.Axe/Selenium.Axe/AxeContextOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Selenium.Axe
+{
+    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
```

Comment:
The existing codebase uses `JsonSerializerSettings` (see `IncludeExcludeManager.cs` L15-20) to apply `NullValueHandling.Ignore` and the camel-casing of property names, rather than doing it per-property/per-class with attributes. Should we be consistent with the existing behavior?

Reply:
i am planning to remove IncludeExcludeManager. we can just do JSON.Serialize(axeContextOptionsObj)

Reply:
but, we could still move this config out like the JsonSerializerSettings